### ISSUE TITLE
Missing icons when the panels can´t communicate with the web server

### ISSUE THREAD

**Issue by acaproni** [IntegratedAlarmSystem-Group/ias-display#84]
I received this email from Erich and filed this ticket:

_I noticed a small bug (maybe) in the IAS GUI. I had this weather station (see attachment) in nice colours on my screen, but then when I disconnected from the VPN, it correctly figured out that the alarms are now invalid, but the necessary icons are not available._

![screen shot 2018-10-11 at 14 17 09](https://user-images.githubusercontent.com/8500519/46804095-26f12280-cd38-11e8-94be-0f0335f48185.png)

**Comment by sfehlandt** [comment]
The description indicates that the application looses connection not only with the webserver but with the whole application. What happens here is that the javascript keeps running but does not receive any Alarm and hence it makes them invalid.

Since the VPN is closed the javascaript has no access to the icons of the invalid Alarms (located in the static files of the ias-display). This is normal behaviour in a web application that looses connection with its static files.

In short, this is not a bug. And hence I am closing it now.

**Comment by acaproni** [comment]
Is there a way to pre-load the icons?
I think we already discussed to show the missing of the conncetion with the web server somewhere in the panels

**Comment by acaproni** [comment]
I agree that with will not be able to do it in a short time; however we keep this ticket open as a todo for the future.

I was thinking that the display could simply turn the menu bar in the upper side in red when the connection is lost.

**Comment by sfehlandt** [comment]
I would open another ticket for that because it is another feature and we would not be fixing the issues with the icons
